topicnet/demos/README.md

```diff
@@ -1 +1,27 @@
-### This folder contains Jupyter notebooks with working  examples of the code
\ No newline at end of file
+# Demo
+This section provides demonstrations of how to use this library in NLP tasks.
+
+1. [RTL-Wiki-Preprocessing](RTL-Wiki-Preprocessing.ipynb) --  notebook describing how to get a Wikipedia dataset and write data in VW format.
```

Comment:
notebook working with a dataset introduced in https://dl.acm.org/doi/10.5555/2984093.2984126. It serves as an example of typical preprocessing pipeline: getting a dataset, lemmatizing it, extracting n-grams / collocations, writing data in VW format.

how to get a dataset from 
, preprocess